### PR TITLE
Homepage features widget styles

### LINKED ISSUE #59: Homepage: Features section
Screen Shot 2020-01-23 at 6 14 46 PM

### PR DESCRIPTION
**Pull request by benlk**
## Changes

This pull request makes the following changes:

- Styles a Largo Recent Posts widget on the homepage bottom, with "Default" widget background set
- turns `.widgettitle` styles into a mixin, and adds logic to resize the text on mobile
- turns the homepage shadow styles into a mixin
- copies in `partials/widget-content.php` from Largo to add the `morelink` link.
- uses a placeholder style for `.btn`, until #71 is merged.

Screen Shot 2020-01-29 at 12 59 05
Screen Shot 2020-01-29 at 12 59 30

## Why

Resolves #59

## Testing/Questions

Features that this PR affects:

- widgettitle styles

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?
- [x] ~This is waiting for the button styles in #71 to be merged, which will cause a merge conflict in `less/mixins.less`~ resolved in 44604d6aee40c75293a0cc577cbddbfc4b47532e
- [x] ~This still needs the blue swooshy background image to be added; I don't have access to that today.~ resolved in e6e3ce8749da99c2dee8e989591b78189f07fbab
- [x] Why is the shadow cut off? 
    Screen Shot 2020-01-29 at 12 57 16
- [x] Why is the button background cut off?
    Screen Shot 2020-01-29 at 12 57 13

Steps to test this PR:

0. Make sure that the three most-recent posts in a category have featured images
1. Add a Largo Recent Posts widget to the Homepage Bottom widget area:
    - title: Features Heading
    - 3 posts
    - avoid duplicates
    - large thumbnail
    - custom excerpt or sentences
    - no byline
    - no publish date
    - no top term
    - limit to that category

### CODE REVIEW

**Comment by benlk** [comment]
This is at time; I anticipate the merge conflicts from #71 and adding the blue swooshy background will take 30 minutes.

**Comment by benlk** [comment]
#71 is merged in and the swoosh has been added as an SVG background. Screenshots have been updated.

Two bugs that I don't understand: the button background is cut off, and the shadows are cut off.

**Comment by joshdarby** [comment]
> Why is the button background cut off?

I think that's happening because the bottom of the grid area is meeting the bottom of the button? (if that makes sense)

Screen Shot 2020-01-30 at 9 37 25 AM

If you add a `@measure` padding to the bottom of that `li` it prevents it from being cut off
Screen Shot 2020-01-30 at 9 37 47 AM

File: wp-content/themes/ipbs-largo/less/mixins.less [hunk 1]
```diff
@@ -1,3 +1,18 @@
+.widgettitle-mixin() {
```
Comment (joshdarby) [inline wp-content/themes/ipbs-largo/less/mixins.less:1]: Can we add a `text-align: center;` to this mixin?

Comment (benlk) [reply wp-content/themes/ipbs-largo/less/mixins.less:1]: Sounds like a good idea.

Comment (benlk) [reply wp-content/themes/ipbs-largo/less/mixins.less:1]: added in https://github.com/INN/umbrella-ipbs/pull/72/commits/093a46fb4bed32d8448b6c268a2a474ceb93226c

**Comment by benlk** [comment]
Added half an em padding-bottom to the element containing the button, and now things are okay. 🤷‍♂  https://github.com/INN/umbrella-ipbs/pull/72/commits/c0df26cb4c0812b0f7bc638923fa8f9cd4b5a2a0

Ready for re-review, I think.

**Comment by joshdarby** [comment]
Screen Shot 2020-01-30 at 5 14 29 PM

@benlk Looks like we need that padding-bottom fix on mobile too. Other than that, I have no other complaints!

**Comment by benlk** [comment]
The btn's problem was for the same reason as the shadow's problem: elements whose display crossed the bounds of the containing `li` were being cut off because of `overflow: hidden;`. In addition, the button here was set to display inline rather than inline-block.

I've also updated previous cases on the homepage of arrows not using `.arrow()` to use that mixin. This affects the homepage top quick links menu and readmore links in this widget.

**Review by joshdarby — APPROVED** [review]

**Comment by benlk** [comment]
Deployed, and a widget has been added to the the homepage http://ipbs.staging.wpengine.com/ . However, with a lack of posts, the widget shows nothing.

**Pull request merged**